.gitignore

```diff
@@ -1,3 +1,5 @@
 /target
 /chrome
 /chromedriver
+/.envrc
+/.direnv
```

Comment:
Why ignore those two?

Reply:
.envrc is a direnv file which automatically starts a dev shell when you open the repo's directory.

.direnv is .envrc's cache, with cached flake inputs and artifacts and stuff

Reply:
Let's include the `.envrc` in the repo as well

Reply:
Added :+1: